prometheus/README.md

```diff
@@ -0,0 +1,41 @@
+# Prometheus Example
+
+This example demonstrates how to use the OpenTelemetry SDK 
+to instrument a simple application using Prometheus as the metric exporter and expose the metrics via HTTP. 
+
+These are collected by a Prometheus instance which is configured to pull these metrics via HTTP. 
+
+# How to run
+
+## Prerequisites
+* Java 1.7
+* Docker 19.03
+
+## 1 - Compile 
+```shell script
+../gradlew shadowJar
+```
+## 2 - Run Prometheus
+
+Start Prometheus instance with a configuration that sets up a HTTP collection job for  ```127.0.0.1:19090```
+
+See [prometheus.yml](prometheus.yml)
+
+```shell script
+docker run --network="host" --rm -it \
+    --name prometheus \
+    -v $(pwd)/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus 
```

Comment:
not necessarily required for this PR, but this doesn't work at all on macs, since docker host networking isn't a thing on macs. We need to get this example re-worked in order to work more generically outside of only linux.

Reply:
👍 opened Issue #11 to address this